composer.json

```diff
@@ -19,7 +19,7 @@
         "drupal/core-composer-scaffold": "^10",
         "drupal/core-dev": "^10",
         "drupal/styleguide": "^2",
-        "drush/drush": "^11.1",
```

Comment:
Let's take the chance to remove cweagans/composer-patches in line 10, we don't have patches declared.